deploy/update-version.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
```

Comment:
This file is and a version.go is duplicated in your other PR:s. Would it be a lot of work to make this code shareable?

Reply:
I'm quite fond of this idea. I'll write up a small RFC on the case because I'm not certain of where to put this code, being a new repo or abusing the wharf-api repo.

Putting the PRs as draft in the meantime

Reply:
Please look now